.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "plugins": [
+    "meteor",
+    "import"
+  ],
+  "extends": [
+    "standard"
+  ],
+  "rules": {
+
+    "callback-return": 2,
+    "arrow-parens": 0,
```

Comment:
just realized I accidentally set these to 0 (off) by mistake, I meant to make them 2 (error). I'm fixing this and will include it in my next PR